content/02-types.md

```diff
@@ -521,7 +521,7 @@ typedef User = {
 <!--label:types-structure-performance-->
 #### Impact on Performance
 
-Using structures and, by extension, [structural subtyping](type-system-structural-subtyping) has no impact on performance when compiling to [dynamic targets](define-dynamic-target). However, on [static targets](define-static-target) a dynamic lookup has to be performed which is typically slower than a static field access.
+Using structures, and by extension, [structural subtyping](type-system-structural-subtyping), has no impact on performance when compiling to [dynamic targets](define-dynamic-target). However, on [static targets](define-static-target) a dynamic lookup has to be performed which is typically slower than a static field access.
```

Comment:
I think this should actually be `… structures and, by extension, structural subtyping, …`

(See e.g. https://www.macmillandictionary.com/dictionary/british/by-extension)

Reply:
Will fix